tsconfig.json

```diff
@@ -20,6 +21,9 @@
       ],
       "typings/*": [
         "src/typings/*"
+      ],
+      "utils/*": [
+        "src/utils/*"
```

Comment:
I disagree with adding all these different absolute paths, what are we really saving? All your doing is saving 

`@/utils` to using `utils`, is that a great saving? Keep absolute paths to a minimum, I'd say just the `@` and no more, get rid of `styles`, `typings` and `utils`. This of course should probably be done in a separate PR, I'm not asking for you to change this now.